style.css

```diff
@@ -1,74 +1,73 @@
-* {
-  color: blue;
-}
+    * {
+    color: blue;
+  }
 
-p {
-  border: 3px;
-  border-style: solid;
-  border-color: #FF0000;
-  padding: 1em;
-  /*background-color: white;*/
-}
+  p {
+    border: 3px;
+    border-style: solid;
+    border-color: #FF0000;
+    padding: 1em;
+  }
 
-.inner {
-  display: inline-block;
- margin-top: 40px;
- margin-right: 75px;
- 
-}
+  .inner {
+    display: inline-block;
+    margin-top: 40px;
+    margin-right: 75px;
+  }
 
-body {
-  background-color: grey;
-}
+  body {
+    background-color: grey;
+  }
 
-.footer {
-  width: 100%;
-  min-height: 132px;
-  background-color: #1F1F1F;
-  text-align: center
-}
+  .footer {
+    width: 100%;
+    min-height: 132px;
+    background-color: #404040;
+    text-align: center
+  }
 
-.footer .fa {
-  display: inline-block;
-  vertical-align: middle;
-  color: #444;
-  font-size: 2.2em;
-  background-color: #fff;
-  border: 4px solid #E6E6FA;
-  border-radius: 48px;
-  height: 52px;
-  line-height: 44px;
-  width: 52px;
-  margin: 20px 5px;
-}
+  .footer .fa {
+    display: inline-block;
+    vertical-align: middle;
+    color: #444;
+    font-size: 2.2em;
+    background-color: #fff;
+    border: 4px solid #E6E6FA;
+    border-radius: 48px;
+    height: 52px;
+    line-height: 44px;
+    width: 52px;
+    margin: 20px 5px;
+  }
 
-.footer {
-  inline-block;
-}
+  .footer {
+    inline-block;
+  }
 
-.copy_text {
-  font-family: Helvetica;
-  font-size: 0.8em;
-  font-style: normal;
-  color: #fff;
-}
+  .copy_text {
+    font-family: Helvetica;
+    font-size: 0.8em;
+    font-style: normal;
+    color: #fff;
+  }
 
-header img {
-  /*float: left;*/
-  margin-top: 15px;
-  margin-right: 30px;
-}
+  .featured_section img {
+    float: right;
+    width: 50%;
+    margin-top: 20px;
+    margin-right: 25px;
+  }
 
-img {
-  <img src="http://placehold.it/300x375/000000" style="height:80%;">
-}
+  .footer {
+    alight: left;
+  }
 
-head .div {
-  inline: block;
-}
-
-header ul li {
-display: inline-block;
-margin-top: 40px;
-margin-right: 75px;
-}
+  @media only screen and (max-width: 501px) {
+    body {
+      background-color: lightblue;
+    }
+  @media only screen and (min-width: 640px) {
+      .icon-menu {
```

Comment:
You are using Font Awesome, so this class will not actually hide your hamburger menu.